api/types/plugins/logdriver/entry.pb.go

```diff
@@ -428,7 +428,7 @@ func skipEntry(dAtA []byte) (n int, err error) {
 }
 
 var (
-	ErrInvalidLengthEntry = fmt.Errorf("proto: negative length found during unmarshaling")
+	ErrInvalidLengthEntry = fmt.Errorf("proto: negative length found during unmarshalling")
```

Comment:
Please do not edit this file, which is automatically generated from `entry.proto`.